app/controllers/propertyLinking/ClaimPropertyRelationshipController.scala

```diff
@@ -118,7 +118,7 @@ class ClaimPropertyRelationshipController @Inject()(
           ))
     }
 
-  def showRelationship(): Action[AnyContent] =
+  def showRelationship(uarn: Option[Long] = None): Action[AnyContent] =
```

Comment:
Do we need this new param? I can't see it used in here

Reply:
good spot 🦅 👁️ Dan .. i meant to remove that